integrated_channels/moodle/models.py

```diff
@@ -83,6 +83,11 @@ class MoodleEnterpriseCustomerConfiguration(EnterpriseCustomerPluginConfiguratio
         )
     )
 
+    transmission_chunk_size = models.IntegerField(
+        default=4,
```

Comment:
I like this default value setup, perhaps we can just default to 1 in canvas (why did we not do that ?)

Reply:
I was wondering that as well and actually went looking at Canvas' models expecting to see this override. I think we just limit it in the client or transmitter?

Reply:
for canvas we rely on the config setting it to 1 (not a good idea!) but the default value would be a good idea I think